src/path.js

```diff
@@ -230,17 +509,18 @@ Path.prototype.draw = function(ctx) {
 /**
  * Convert the Path to a string of path data instructions
  * See http://www.w3.org/TR/SVG/paths.html#PathData
- * @param  {number} [decimalPlaces=2] - The amount of decimal places for floating-point values
+ * @param  {object|number} [options={decimalPlaces:2, optimize:true}] - Options object (or amount of decimal places for floating-point values for backwards compatibility)
  * @return {string}
  */
-Path.prototype.toPathData = function(decimalPlaces) {
-    decimalPlaces = decimalPlaces !== undefined ? decimalPlaces : 2;
+Path.prototype.toPathData = function(options) {
+    // set/merge default options
+    options = createSVGOutputOptions(options);
```

Comment:
no need for a separate function, I think this wold also work
```js
Path.prototype.toPathData = function(options = 2) {
  const def = { decimalPlaces: 2, optimize: true, flipY: true};
  options = Object.assign(def, isNaN(options) ? options : {decimalPlaces:options});
}
```

Reply:
My code was structured differently at first, and I wanted to be able to use the options in the glyph.js wrapper functions as well, in order to pass the base value for flipY from the glyph metrics and already have the default values there. But I didn't want to make things more complicated with this PR as they already are. We might need that in the future though, that's why I left them in a function.